packages/fleet/lib/models/deployments.ts

```diff
@@ -28,7 +27,7 @@ const DBDeployment = {
         return {
             id: deployment.id,
             image: deployment.image,
-            commit_id: deployment.commitId,
+            commit_id: '0000000000000000000000000000000000000000' as CommitHash, //TODO: remove
```

Comment:
keeping the `commit_id` column to avoid issue when deploying. Old jobs instances would still relies on its existence. Once this commit is deployed and no running code relies on this column I will add a migration and cleanup the code